src/screens/FeedbackScreen.js

```diff
@@ -121,6 +121,11 @@ export const FeedbackScreen = () => {
                     textAlignVertical="top"
                   />
                   <CheckBox
+                    accessibilityLabel={`${
+                      consent
+                        ? texts.accessibilityLabels.checkbox.active
+                        : texts.accessibilityLabels.checkbox.inactive
+                    } Ich bin mit dem Speichern meiner Daten einverstanden.`}
```

Comment:
can you please move the text to `texts`? and here a well, what about brackets around one part of the label for better voice over reading?

Reply:
I added it to the `texts` file as `feedbackScreen.checkboxTitle`

https://github.com/smart-village-solutions/smart-village-app-app/commit/f425b1ee2acfc59d90979d9e08ec3fa64b692c06